## Atividade Final - Simulação 

In [14]:
!pip install -q openpyxl
!pip install -q scikit-learn
import pandas as pd
from sklearn.linear_model import LinearRegression

In [5]:
df1 = pd.read_excel('dados1.xlsx')

In [6]:
df1.head()

Y     X1    X2
0  74.8  15.31  57.3
1  74.0  15.20  63.8
2  72.9  16.25  65.4
3  70.0  14.33  57.0
4  74.9  14.57  63.8

In [7]:
# Calcular a matriz de correlações
correlation_matrix = df1.corr()

In [9]:
#Letra (a) matriz de correlação
print(correlation_matrix)

           Y        X1        X2
Y   1.000000  0.913319  0.851384
X1  0.913319  1.000000  0.925711
X2  0.851384  0.925711  1.000000


b)

A correlação entre Y (Preço de vendas) e X1 (Preço de vendas) é de 0.913319. Essa correlação é bastante forte e positiva, o que indica que existe uma relação linear positiva entre essas duas variáveis. Isso significa que, em geral, à medida que o preço de vendas aumenta, o preço de vendas também tende a aumentar.

A correlação entre Y (Preço de vendas) e X2 (Valor estimado) é de 0.851384. Essa correlação também é considerada forte e positiva, indicando uma relação linear positiva entre as duas variáveis. Isso implica que, em geral, à medida que o valor estimado aumenta, o preço de vendas também tende a aumentar.

A correlação entre X1 (Preço de vendas) e X2 (Valor estimado) é de 0.925711. Mais uma vez, essa correlação é forte e positiva, sugerindo uma relação linear positiva entre as variáveis. Portanto, quando o preço de vendas aumenta, o valor estimado também tende a aumentar.

Com base nessas informações, podemos concluir que existe uma correlação positiva e forte entre o preço de vendas (Y), o preço de vendas anterior (X1) e o valor estimado (X2). Isso sugere que essas variáveis estão relacionadas linearmente e têm influência uma sobre a outra.